src/Components/FeelingRoom/fineAnimations.js

```diff
@@ -13,23 +13,23 @@ export const fineBackgroundAnimation = () => {
 }
 
 export const floatAnimation = (element, scale) => {
-  let floatTl = gsap.timeline({repeat: 200, });
+  let floatTl = gsap.timeline({repeat: 200, yoyo:true});
```

Comment:
This is a great solution for getting the bubbles to gather again without being so jarring. Nice work